BUILD

```diff
@@ -418,7 +418,9 @@ cc_test(
     copts = COPTS,
     data = [
         ":test_plugin",
-    ],
+    ] + glob([
+        "src/google/protobuf/**/*",
```

Comment:
Some EncodeDecode tests needs proto files. Just include all source files for simplicity.